client/src/components/NewLayout/SearchBox.tsx

```diff
@@ -5,8 +5,10 @@ import { usePlaceMethod, usePlaceState } from '../../contexts/StateContext'
 import icons from '../../utils/icons'
 import { InputGroup } from '../../styles/styles'
 import { MOON_BLUE } from '../../utils/CONSTANTS'
+import { useI18n } from '../../contexts/I18nProvider'
 
 const SearchBox = () => {
+  const t = useI18n().translation.components.search_box
```

Comment:
Styling wise I would  prefer ```useI18n(x => x.translation.components.search_box)```. There isn't a good practicle  reason in this cirmunstace but does make it easier to build functional abstractions.